experiments/monaco/README.md

```diff
@@ -0,0 +1,3 @@
+# JupyterLab Benchmarks
+
+A page with may prosemirror editors.
```

Comment:
Should this be monaco editors instead of prosemirror?

Reply:
Fixed.